src/dev/build/tasks/create_archives_task.js

```diff
@@ -71,5 +81,20 @@ export const CreateArchivesTask = {
           throw new Error(`Unexpected extension for archive destination: ${destination}`);
       }
     }
+
+    const reporter = CiStatsReporter.fromEnv(log);
+    if (reporter.isEnabled()) {
+      await reporter.metrics(
+        await Promise.all(
+          files.map(async path => {
+            return {
+              name: 'distributable size',
+              subName: Path.basename(path),
+              value: (await asyncStat(path)).size,
```

Comment:
Yeah, the basename of each archive is unique and includes things like the os, architecture, and archive format. We currently only create archives for the current platform on CI but the change in size should be pretty consistent between the distributable for each platform so I'm not particularly concerned.

Reply:
I wonder if we should standardize on some formats, ala ECS. For instance, version and platform would be easier to search for in this instance than having to figure out what the filename is. It also allows for us to apply a filter to a dashboard.

Reply:
I don't think searching by version or platform will be very useful here though. And I'm not opposed to using a standardized format, but ECS doesn't feel appropriate for the intermediate representation we're creating here.

If anything I think we might want our groups to be
```
${oss ? 'oss ' : ''}distributable size
```
and our metric ids to be the archive type. Version/branch information is really a property of the job and less a property of the metric.

The way I think of the data we're collecting here is how it will look as a table formatted as:

> ### ${group}
> |  id   |  value   | Change from {mergeBase} |
> | --- | ---:| ---:|
> | ${id} | ${valueAsMB} | ${difference} |

Once we're done with getting tables like that reporting on PRs we will work on putting this data into a format compatible with searching/filtering by summarizing the data into a single document and then indexing it into kibana-stats. With that it will be easy to have something like a chart of the average "distributable size" broken down by archive type, base branch, and spreading it out over time. That said, slower and longer term reporting/trends is something we plan to tackle after we get reporting into each PR.

Reply:
Implemented alternate metric group/id in https://github.com/elastic/kibana/pull/65408/commits/5e122d2472cb9cf65944dedc3cb177233dee7188

Reply:
Sync'd with Spencer on this - the portion I was missing was the summary document will contain the version/etc. 👍